docs/user-guide/system-admin/configuration/infinispan.md

```diff
@@ -12,14 +15,38 @@ The Infinispan configuration will be located inside the following module in `sta
 
 Keep in mind that the module version may vary depending on your JBoss version.
 
-### Configuration for Internal data caches
+### Hibernate Cache
+
+The following is the recommended configuration for the Hibernate cache:
+
+```xml
+...
+<cache-container name="hibernate" default-cache="local-query" jndi-name="java:jboss/infinispan/container/hibernate" start="EAGER" module="org.jboss.as.jpa.hibernate:4">
```

Comment:
Well, the strange thing is that the default standalone.xml in eap 6.4.6
does not include the jndi-name attribute for the 'hibernate'
cache-container.

But I don't see a need for Hibernate to use our packaged version of
Infinispan: that's just for our own caches (ie stats caches).  Perhaps we
just need to remove hibernate.cache.region.factory_class from
persistence.xml, and let the platform do its thing.

On 14 March 2016 at 10:29, Carlos Munoz <notifications@github.com> wrote:

> In docs/user-guide/system-admin/configuration/infinispan.md
> <https://github.com/zanata/zanata-server/pull/1111#discussion_r55946974>:
>
> > @@ -12,14 +15,38 @@ The Infinispan configuration will be located inside the following module in `sta
> >
> >  Keep in mind that the module version may vary depending on your JBoss version.
> >
> > -### Configuration for Internal data caches
> > +### Hibernate Cache
> > +
> > +The following is the recommended configuration for the Hibernate cache:
> > +
> > +```xml
> > +...
> > +<cache-container name="hibernate" default-cache="local-query" jndi-name="java:jboss/infinispan/container/hibernate" start="EAGER" module="org.jboss.as.jpa.hibernate:4">
>
> Well, it does work out of the box, but you have to tell it that a) you
> want it to use infinispan as the cache provider (i.e. use the
> InifnispanRegionFactory), and 2) where is the inifinispan cache you want to
> use (i.e. jndi)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1111/files#r55946974>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat